test/UserTest.php

```diff
@@ -10,13 +10,22 @@ public function testSuccessCreateUserCardHtml1()
     {
         $user = new User(1,'fakeUsername', 'fakeHashedPass', 'fake bio', 'fakeAvatar.jpeg');
         $result = $user->createUserCardHtml();
-        $this->assertEquals('<a class="profileLink" href="./userProfile.php?username=fakeUsername"><div class="userCard"><img class="avatarImg" src="./assets/imgs/avatars/fakeAvatar.jpeg" alt="Profile Picture"><p class="cardUsernameText">fakeUsername</p></div></a>', $result);
+        $expected = '<a class="profileLink" href="./userProfile.php?username=fakeUsername"><div class="userCard"><img class="avatarImg" src="./assets/imgs/avatars/fakeAvatar.jpeg" alt="Profile picture" /><p class="cardUsernameText">fakeUsername</p></div></a>';
+        $this->assertEquals($expected, $result);
     }
 
     public function testSuccessSanitizedBio()
```

Comment:
should be called `testSuccessGetSanitizedBio`

Reply:
and also you should incorporate some `\n`s into your input string and expect them to turn into `<br>`s